src/main/scala/is/hail/linalg/BlockMatrix.scala

```diff
@@ -1345,3 +1439,151 @@ class WriteBlocksRDD(path: String,
     blockPartFiles.iterator
   }
 }
+
+case class ExportRectanglesRDDPartition(index: Int, rectangle: Array[Long]) extends Partition {
+  require(rectangle.length == 4)
+}
+
+class ExportRectanglesRDD(hc: HailContext,
```

Comment:
I don't totally understand why this needs to be an RDD -- it seems like it's only used to write to files. In that case, a mapPartitions operation could do the trick. Is that right?

Reply:
mapPartitions on what RDD?

Reply:
parallelize the array of file names?

Reply:
Done